### ISSUE TITLE
i18n: translate new UPC language keys from PR #1075

### ISSUE THREAD

**Issue by newlinewebdesign** [j2commerce/j2commerce#1076]
## Background

[PR #1075](https://github.com/j2commerce/j2commerce/pull/1075) (merged 2026-05-23) added variant UPC AJAX update parity with SKU/price. The PR introduced new language keys in `en-US` and `en-GB`, but the other 19 supported languages have not yet been updated.

### Admin (`administrator/components/com_j2commerce/language/<lang>/com_j2commerce.ini` AND `administrator/language/<lang>/com_j2commerce.ini`)

```ini
COM_J2COMMERCE_CONFIG_SHOW_UPC_GUEST="Show UPC To"
COM_J2COMMERCE_CONFIG_SHOW_UPC_GUEST_DESC="Who can see product UPCs."
COM_J2COMMERCE_PRODUCT_SHOW_PRODUCT_UPC_LABEL="UPC"
COM_J2COMMERCE_PRODUCT_LIST_SHOW_PRODUCT_UPC_LABEL="UPC (List View)"
```

### Site (`components/com_j2commerce/language/<lang>/com_j2commerce.ini`)

```ini
COM_J2COMMERCE_PRODUCT_SHOW_PRODUCT_UPC_LABEL="UPC"
COM_J2COMMERCE_PRODUCT_LIST_SHOW_PRODUCT_UPC_LABEL="UPC (List View)"
COM_J2COMMERCE_PRODUCT_UPC="UPC"
COM_J2COMMERCE_UPC="UPC:"
```

## Scope

Translate the above keys into all 19 non-English locales using the `joomla-language` skill (DeepL + Lara MCP).

**Rules:**
- Use formal tone (`formality: "more"`).
- Preserve ALL diacritics exactly as produced by DeepL.
- "UPC" is an acronym (Universal Product Code) — keep as `UPC` in most languages; localize only the surrounding label/help text.
- Mirror admin strings to BOTH `administrator/components/com_j2commerce/language/<lang>/` AND `administrator/language/<lang>/` (per CLAUDE.md dual-mirror rule).

## Files To Update (per language)

- `administrator/components/com_j2commerce/language/<lang>/com_j2commerce.ini`
- `administrator/language/<lang>/com_j2commerce.ini`
- `components/com_j2commerce/language/<lang>/com_j2commerce.ini`

## Acceptance

- All 19 locales contain the 4 admin keys + 4 site keys.
- Diacritics intact (`Überprüfung`, `Configuración`, etc.).
- Translated `.ini` files load without parser errors (no unescaped quotes).
- Spot-check rendering in Joomla 6 admin and frontend.

## Reference

- Merge commit: `a4adc86d1506`
- PR: #1075
- PRD: `docs/prd/upc-ajax.md`

**Comment by newlinewebdesign** [comment]
Fix submitted in PR #1078 — 36 files, 18 locales, +144 lines additive. fa-IR skipped (no main `com_j2commerce.ini` in repo). Translation via DeepL formal tone, UPC kept as Latin acronym for codebase consistency.
